SPECS/envoy/envoy.spec

```diff
@@ -15,92 +15,53 @@
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
-
+%global __strip /bin/true
```

Comment:
Why? This could use a comment with justification.

Reply:
Fixed. Added comment in the spec file